### DISCUSSION TITLE
Why some react-table examples use div tags instead of table/thead/tbody/tr/th/td?

### DISCUSSION THREAD

**Discussion by leodevbro** [TanStack/table#3669] [Q&A]
Hello, as I see, in `react-table` examples, some codesandbox examples use `table/thead/tbody/tr/th/td` tags like this sandbox:

https://react-table-v7.tanstack.com/docs/examples/basic

But some examples use just `div` tags:

https://react-table-v7.tanstack.com/docs/examples/column-resizing

Which approach is better if I want to build very responsive and very feature-rich/advanced table? And why?
I mentioned the word "responsive ", yeah, I know `react-table` (headless) does not do CSS design/responsive styles, it's a tabling engine, but I think the decision between these two (`div` or `table/thead/tbody/tr/th/td`) approaches will effect the CSS styling experience.

**Comment by marcos-oliveira** [comment]
I found that for v8 there is a way to resize table tags: https://github.com/TanStack/table/blob/main/examples/react/column-sizing/src/main.tsx

but I didn't find it for v7, is it possible?

**Comment by naveen1337** [comment]
Is there any advantage here to choose div tags over table html tags.
